pandas/tests/indexes/categorical/test_category.py

```diff
@@ -976,12 +976,6 @@ def test_engine_type(self, dtype, engine_type):
         assert np.issubdtype(ci.codes.dtype, dtype)
         assert isinstance(ci._engine, engine_type)
 
-    def test_getitem_2d_deprecated(self):
-        # GH#30588 multi-dim indexing is deprecated, but raising is also acceptable
-        idx = self.create_index()
-        with pytest.raises(ValueError, match="cannot mask with array containing NA"):
-            idx[:, None]
```

Comment:
This was removed because the base class version (which checks for the deprecation) now passes (since I added the deprecation warning)